src/Interpreters/ProcessList.cpp

```diff
@@ -11,6 +11,7 @@
 #include <Common/typeid_cast.h>
 #include <Common/Exception.h>
 #include <Common/CurrentThread.h>
+#include "Parsers/IAST.h"
```

Comment:
Consider to avoid raw includes `#include <Parsers/IAST.h>` can be specified as other includes.